src/test/java/se/liu/ida/hefquin/engine/federation/access/impl/Neo4jResponseParserTest.java

```diff
@@ -0,0 +1,88 @@
+package se.liu.ida.hefquin.engine.federation.access.impl;
+
+import org.junit.Test;
+import se.liu.ida.hefquin.engine.EngineTestBase;
+import se.liu.ida.hefquin.engine.federation.Neo4jServer;
+import se.liu.ida.hefquin.engine.federation.access.FederationAccessException;
+import se.liu.ida.hefquin.engine.federation.access.Neo4jRequest;
+import se.liu.ida.hefquin.engine.federation.access.RecordsResponse;
+import se.liu.ida.hefquin.engine.federation.access.impl.iface.Neo4jInterfaceImpl;
+import se.liu.ida.hefquin.engine.federation.access.impl.req.Neo4jRequestImpl;
+import se.liu.ida.hefquin.engine.federation.access.impl.reqproc.Neo4jRequestProcessor;
+import se.liu.ida.hefquin.engine.federation.access.impl.reqproc.Neo4jRequestProcessorImpl;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.RecordEntry;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.TableRecord;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.*;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.CypherVar;
+
+import java.util.Iterator;
+
+import static org.junit.Assert.*;
+
+public class Neo4jResponseParserTest extends EngineTestBase {
+
+    @Test
+    public void parseEdgeAndNodesTest() throws FederationAccessException {
+        if (!skipLiveWebTests) {
```

Comment:
Instead of using this flag, please introduce another one, which may be called `skipLocalNeo4jServerTests`.

Reply:
Also, for these tests you should somehow document (inside our source code) what the LPG is that is expected to be in the Neo4j instance that you are using here.